internal/weldr/api.go

```diff
@@ -1482,14 +1490,16 @@ func (api *API) composeInfoHandler(writer http.ResponseWriter, request *http.Req
 	reply.Deps = Dependencies{
 		Packages: make([]map[string]interface{}, 0),
 	}
-	reply.ComposeType = compose.OutputType
-	reply.QueueStatus = compose.QueueStatus
-	if compose.Image != nil {
-		reply.ImageSize = compose.Size
+	// Weldr API assumes only one image build per compose, that's why only the
+	// 1st build is considered
+	reply.ComposeType, _ = compose.ImageBuilds[0].ImageType.ToCompatString()
+	reply.QueueStatus = compose.GetState().ToString()
+	if compose.ImageBuilds[0].Image != nil {
+		reply.ImageSize = compose.ImageBuilds[0].Image.Size
```

Comment:
Oh, one more bad rebase bug. The line should be:
```
reply.ImageSize = compose.ImageBuilds[0].Size
```

*Note: actually the condition doesn't make much sense but please don't remove it because it's a bit of unrelated change and I already did it in one of my yet-to-be-posted PR.*

Reply:
thanks!